core/fokia/provisioner.py

```diff
@@ -540,6 +602,8 @@ def check_all_resources(self, quotas, **kwargs):
                                           ip_request=args.ip_request,
                                           network_request=args.network_request,
                                           project_name=args.project_name)
-    # print(response)
-    # print(provisioner.get_cluster_details())
-    # print(provisioner.get_private_key())
+    print(provisioner.get_cluster_details())
```

Comment:
I use this part of the code for local testing,it will all be removed in the future